python/arcticdb/version_store/library.py

```diff
@@ -1168,15 +1218,28 @@ def sort_and_finalize_staged_data(
         mode: Optional[StagedDataFinalizeMethod] = StagedDataFinalizeMethod.WRITE,
         prune_previous_versions: bool = False,
         metadata: Any = None,
+        drop_staged_data_on_failure: bool = False
     ) -> VersionedItem:
         """
-        sort_merge will sort and finalize staged data. This differs from `finalize_staged_data` in that it
+        ``sort_and_finalize_staged_data`` will sort and finalize staged data. This differs from `finalize_staged_data` in that it
```

Comment:
"``sort_and_finalize_staged_data`` will sort and finalize staged data"
I know you didn't add this, but let's lose the tautology